### PR TITLE
fix warp normalization to maintain monotonic warps

### PR DESCRIPTION
**Pull request by ahwillia**
**Before**

image

**After**

image

### CODE REVIEW

File: twpca/warp.py [hunk 1]
```diff
@@ -70,12 +70,10 @@ def generate_warps(n_trials, n_timesteps, shared_length, warptype, init, origin_
...
 
-    tau = tau_scale[:, None] * tf.cumsum(d_tau, 1) + tau_shift[:, None]
-
-    # renormalize the warps so that they average to the identity line across trials.
-    #   i.e. mean(tau[:,t]) = t
-    linear_ramp = tf.range(0, shared_length, dtype=tf.float32)
-    tau = tau - tf.reduce_mean(tau, axis=0, keep_dims=True) + linear_ramp[None, :]
+    raw_tau = tau_scale[:, None] * tf.cumsum(d_tau, 1) + tau_shift[:, None]
+    mean_intercept = tf.reduce_mean(raw_tau[:, 0])
+    mean_slope = tf.reduce_mean(raw_tau[:, -1] - raw_tau[:, 0]) / n_timesteps
```
Comment (poolio) [inline twpca/warp.py]: the denominator should be `n_timesteps - 1`

**Review by poolio — CHANGES_REQUESTED** [review]

File: twpca/warp.py (hunk 1 above)
Comment (ahwillia) [reply twpca/warp.py]: 👍
